src/arrayMethodSort.js

```diff
@@ -5,7 +5,36 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let toRepeat = true;
+
+    const bubbleSort = () => {
+      const defaultCheck = (a, b) => {
+        if (String(a) > String(b)) {
```

Comment:
Here is a good situation to use ternary operator [link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)